### ISSUE TITLE
Pokémon Glazed corrupts Twilight Menu

### ISSUE THREAD

**Issue by AozakiWr** [Gericom/GBARunner2#122]
I tried to start Pokémon Glazed (latest version), but after the Gba Splash Screen, strange noises happen on white screen and TwilightMenu files corrupt.

**Comment by RocketRobz** [comment]
I assume you're on a very old version that corrupts the SD card.
Please update TWiLight Menu++.

**Comment by AozakiWr** [comment]
I did the TwilightMenu update, which prevented data corruption, however, the game continues on white screen making strange noises and not starting. Remembering that Pokémon Glazed is a Rom derived from Emerald with a different Save Type.

**Comment by zenseii** [comment]
Please correct me on this but it seems Pokemon Glazed is 32MB. If that is the case then that's your reason for it not running on the DSi. Not sure if this can ever be worked around.

**Comment by AozakiWr** [comment]
The Save Type is Flash 128kb, if that's what you ask.

**Comment by zenseii** [comment]
No, basically the game is too large for the DSi at the moment, if I'm not mistaken. Any code that lies above 16MB will not run because that's the amount of RAM available on the DSi for this purpose sadly.

**Comment by AozakiWr** [comment]
Now I understand. Yes, unfortunately it is 32MB. I was hoping to play on my DSi...

**Comment by zenseii** [comment]
You should try to see if changing around with the settings get you any further. In other words try turning off wram i-cache, setting NTR 67mhz clock speeds in twilightmenu++ settings etc. Also make sure to have disabled vram boost and set it to DS mode and not DSi in twilightmenu++ settings.

**Comment by zenseii** [comment]
@AozakiWr this romhack might be playable now on DSi using this build which added support for larger roms: https://github.com/Gericom/GBARunner2/releases/tag/v20200812-130512_d5dc8d8
If you test it could you please report back?
